src/sections/Home/Stewards-of-industry/index.js

```diff
@@ -1,22 +1,43 @@
 import React from "react";
-
 import BannerSectionWrapper from "./stewards.style";
 import Button from "../../../reusecore/Button";
+import { graphql, useStaticQuery } from "gatsby";
+import { getImage } from "gatsby-plugin-image";
+import { BgImage } from "gbimage-bridge";
 
 const StewardsOfIndustry = () => {
+  const { backgroundImage123 } = useStaticQuery(
+    graphql`
+      query {
+        backgroundImage123: file(
+          relativePath: { eq: "stewards/stewards.png" }
```

Comment:
Since the image is now loaded from the `assets` folder, the `images` folder under the `Stewards-of-industry can be deleted.

Reply:
> Since the image is now loaded from the `assets` folder, the `images` folder under the `Stewards-of-industry can be deleted.

Alright. Will do that.

Reply:
Also, why is there a merge conflict? I just checked and there is no package with a conflicting version to prevent a merge. Any suggestions?

Reply:
Because of the indentation update.....just fetch latest changes and rebase, that would do

Reply:
> Because of the indentation update.....just fetch latest changes and rebase, that would do

I am getting the conflict because of the new package you installed `react-countdown` I have tried to rebase but during merge, it shows the conflict again and I am only seeing this conflict in the `package.json` file

Reply:
How do you mean to add my code, I tried to manually add my packages to the `package.json` file merge seems not to work. I will keep trying it though.

Reply:
Removing the folder is still left.